db/migrate/20210810102028_create_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.references :role, null: false, foreign_key: true
+      t.string :email
+      t.string :password
```

Comment:
I don't think that having passwords in plain text is considered a good practice.